sql/core/src/main/scala/org/apache/spark/sql/DataStreamWriter.scala

```diff
@@ -84,6 +84,17 @@ final class DataStreamWriter private[sql](df: DataFrame) {
   }
 
   /**
+   * Specifies a name to the [[ContinuousQuery]] to be started. This name must be unique among
```

Comment:
DataStreamWriter is not a ContinuousQuery. I think looking at the scala doc page of DataStreamWriter, the phrase "this ContinuousQuery" is confusing.